tests/integration/test_end_to_end.py

```
@@ -0,0 +1,400 @@
+        # Verify mock calls
+        mock_pr.create_review_request.assert_called_once()
+        mock_pr.merge.assert_called_once_with(merge_method="squash")
+
+    async def test_error_recovery_workflow(self):
+        """Test error recovery in workflows."""
+        # Test webhook error recovery
+        webhook_handler = WebhookHandler()
+
+        error_count = 0
+
+        async def failing_handler(event):
```

Ruff (0.13.1)

Unused function argument: `event`

(ARG001)